src/utils/flatten.js

```diff
@@ -28,7 +29,11 @@ export const objToCss = (obj: Object, prevKey?: string): string => {
  */
 const isFalsish = chunk => chunk === undefined || chunk === null || chunk === false || chunk === '';
 
-export default function flatten(chunk: any, executionContext: ?Object, styleSheet: ?Object): any {
+export default function flatten(
+  chunk: any,
+  executionContext: ?Object,
+  styleSheet: ?Object = StyleSheet.master
```

Comment:
This is the primary change of the PR, assigning default back to `StyleSheet.master`